modules/apps/frontend-css/frontend-css-variables-web/src/main/java/com/liferay/frontend/css/variables/web/internal/servlet/taglib/ScopedCSSVariablesTopHeadDynamicInclude.java

```diff
@@ -105,21 +106,22 @@ private void _writeCSSVariables(
 		for (ScopedCSSVariables scopedCSSVariables :
 				scopedCSSVariablesCollection) {
 
+			printWriter.print(StringPool.TAB);
 			printWriter.print(scopedCSSVariables.getScope());
 			printWriter.print(" {\n");
 
 			Map<String, String> cssVariables =
 				scopedCSSVariables.getCSSVariables();
 
 			for (Map.Entry<String, String> entry : cssVariables.entrySet()) {
-				printWriter.print("--");
+				printWriter.print("\t\t--");
```

Comment:
Why do we use `StringPool.TAB` above but not in here?

Reply:
[This](https://github.com/wincent/liferay-portal/pull/354#discussion_r440680796)?

Reply:
Because we use the constants when the character appears alone, but string otherwise :shrug: . The rationale for `StringPool` was to avoid in memory duplication of similar strings. In early Java versions this was an important problem. Then some improvements where made to the compiler which fixed this. And, AFAIK, today you can still make it a bit better than using the compiler alone with `StringPool` (corner cases).

Reply:
![okay](https://user-images.githubusercontent.com/905006/85574851-65775f80-b637-11ea-828a-458bebe15e16.gif)